src/Sulu/Bundle/SecurityBundle/Admin/SecurityAdmin.php

```diff
@@ -107,12 +110,6 @@ public function getSecurityContexts()
                         PermissionTypes::EDIT,
                         PermissionTypes::DELETE,
                     ],
-                    static::GROUP_SECURITY_CONTEXT => [
```

Comment:
as there is not interface for using the group functionality, these permissions do not have any consequence in the administration interface in Sulu 2